pages/learn/develop/integrations/aws.md

```diff
@@ -208,15 +210,15 @@ where you need to replace `KEYFILE` with the relevant filename (such as `xxxxxxx
 
 ### Shortcuts:
 
-* [AWS IoT console](https://console.aws.amazon.com/iot/home)
-* [AWS IoT Documentation](https://aws.amazon.com/documentation/iot/)
-* [AWS IoT API Reference](http://docs.aws.amazon.com/iot/latest/apireference/Welcome.html)
-* [AWS SDK reference](https://aws.amazon.com/tools/#sdk), list of SDKs in all supported languages
-* [AWS IoT MQTT Client](https://console.aws.amazon.com/iot/home#/mqtt), useful for debugging AWS IoT communication
+- [AWS IoT console](https://console.aws.amazon.com/iot/home)
+- [AWS IoT Documentation](https://aws.amazon.com/documentation/iot/)
+- [AWS IoT API Reference](http://docs.aws.amazon.com/iot/latest/apireference/Welcome.html)
+- [AWS SDK reference](https://aws.amazon.com/tools/#sdk), list of SDKs in all supported languages
+- [AWS IoT MQTT Client](https://console.aws.amazon.com/iot/home#/mqtt), useful for debugging AWS IoT communication
 
 ### Sample Apps
 
 A few sample apps to get started:
 
-* [balena-aws-lambda]({{ $links.githubPlayground }}/balena-aws-lambda) and [balena-aws-device]({{ $links.githubPlayground }}/balena-aws-device)
-* [balenaCloud AWS IoT MQTT Broker Example]({{ $links.githubPlayground }}/balena-aws-iot-mqtt-example) with full blog post for example:
+- [balena-aws-lambda]({{ $links.githubPlayground }}/balena-aws-lambda) and [balena-aws-device]({{ $links.githubPlayground }}/balena-aws-device)
```

Comment:
I believe these should be moved from `-playground` to `-examples` and updated accordingly.

Reply:
yes, they have been moved. GitHub does the redirect automatically. should we change it here too?

Reply:
I think that's best. I added the link stub here: https://github.com/balena-io/docs/pull/1445